README.md

```diff
@@ -70,43 +72,60 @@ RC4(<JSON_SERIALIZED_OPTIONAL_FOOTER>)
 
 ###  Mandatory Footer (28 Bytes)
 
-CaRT ends its file with a mandatory footer which allow the format to read the footer and return the hashes 
+CaRT ends its file with a mandatory footer which allow the format to read the footer and return the hashes
 without reading the whole file
 ```
  4s      QQ           Q
 TRAC<RESERVED><OPT_FOOTER_LEN>
 ```
 
-## Command line interface 
+## Command line interface
 
-By installing the pip package, you get access to the CaRT library and also access to the CaRT CLI. 
+By installing the pip package, you get access to the CaRT library and also access to the CaRT CLI.
 
 The CaRT CLI has the following priority for its options:
 
 * There are defaults values for all the options inside the CLI
-* Default values are overridden by options in `~/.cart/cart.cfg` 
+* Default values are overridden by options in `~/.cart/cart.cfg`
 * Values in the configuration file are overridden by CLI options
 
 These are the options available in the CaRT CLI:
 
 ```
-Usage: cart [options] file1 file2 ... fileN
+usage: cart [options] file1 file2 ... fileN
+
+            The CaRT file format is used to store/transfer malware and its associated metadata.
+
+            It neuters the malware so it cannot be executed and encrypts it so anti-virus software
+            cannot flag the CaRT file as malware.
+
+
+positional arguments:
+  file
 
-Options:
-  --version             show program's version number and exit
+options:
   -h, --help            show this help message and exit
+  -v, --version         show program's version number and exit
+  -d, --delete          Delete original after operation succeeded
   -f, --force           Replace output file if it already exists
   -i, --ignore          Ignore RC4 key from conf file
-  -j JSONMETA, --jsonmeta=JSONMETA
+  -j JSONMETA, --jsonmeta JSONMETA
                         Provide header metadata as JSON blob
-  -k KEY, --key=KEY     Use private RC4 key (base64 encoded). Same key must be
-                        provided to unCaRT.
+  -k KEY, --key KEY     Use private RC4 key (base64 encoded). Same key must be provided to unCaRT.
   -m, --meta            Keep metadata around when extracting CaRTs
-  -n FILENAME, --name=FILENAME
+  -n FILENAME, --name FILENAME
                         Use this value as metadata filename
-  -o OUTFILE, --outfile=OUTFILE
+  -o OUTFILE, --outfile OUTFILE
                         Set output file
   -s, --showmeta        Only show the file metadata
+
+            CaRT is smart enough to determine if a file needs to be CaRTed or unCaRTed.
+
```

Comment:
Is there really newlines between the lines?

Reply:
yes!